### DISCUSSION TITLE
Fix mention notifications for users with an underscore in their username

### DISCUSSION THREAD

**Discussion by akutch** [deso-protocol/dips#54] [Medium CIPs]
_Please feel free to close if there's already an open issue tracking this or if this is a know issue._

## The problem
It seems that users with an underscore (`_`) in their name are not receiving mention ($/@) notifications. 

As brought up in [this post](https://bitclout.com/posts/7280df5b7539503ec0878efeea806110adae929abf4c464539391378ab4cc229)

<img width="600" alt="Screen Shot 2021-07-13 at 7 18 13 AM" src="https://user-images.githubusercontent.com/6456722/125443090-9c2a51fa-121e-4a3d-a3ca-b60c738513b3.png">

## Proposed solution

**Idea 1:** 
Track down what's causing the notifications to get missed for `_` users, and start surfacing them on the notifications page

**Idea 2:** 
Disallow users from having a `_` in their username 😬

**Comment by ItsAditya-xyz** [comment]
Disagreed with removal of "_" in usernames.

This problem is solved by the way. Wait for latest deployment.

Check : https://github.com/bitclout/core/pull/61

**Reply by akutch** [reply]
Option 2 was a bit of a joke! 

They’re so on top of it 😇 I’ll close this out.

**Comment by akutch** [comment]
Hm, how do I close this out?

**Comment by lazynina** [comment]
this is now resolved! I'm going to lock the discussion.
